### ISSUE TITLE
`drush entup --no-cache-clear` is clearing cache

### ISSUE THREAD

**Issue by hanoii** [drush-ops/drush#3586]
I am looking into running this command to `detect` if there are pending updates under certain circumstances, and I don't want to clear cache to make it faster but it's always clearing.

drush 9.3.0

**Comment by keithdoyle9** [comment]
Same issue is happening to me.

```
drush entup -y -v --uri=http://example.com --no-cache-clear
 [info] Executing: mysql --defaults-file=/tmp/drush_hvzTLG --database=db --host=dbhost --port=3306 --silent  < /tmp/drush_uR9iVs
 [info] Executing: mysql --defaults-file=/tmp/drush_2NPKM8 --database=db --host=dbhost --port=3306 --silent  < /tmp/drush_TPljMU
 [success] No entity schema updates required
 [info] /usr/local/bin/drush  --backend=2 --verbose --root=/var/lib/jenkins/workspace/ --uri=http://example.com  cache-rebuild 2>&1
 [success] Cache rebuild complete.
 [success] Finished performing updates.
```

**Comment by weitzman** [comment]
Fixed in ab8e8a7c1. 

Detection of pending updates should be handled by `updatedb:status`. To pick out just entity updates you can use grep, or --format=json and some more processing. It will require no further processing once we have #3718 (and follow-ups).
